tests/kafkatest/services/streams.py

```diff
@@ -465,6 +468,9 @@ def prop_file(self):
         properties['reduce.topic'] = self.REDUCE_TOPIC
         properties['join.topic'] = self.JOIN_TOPIC
 
+        # Long.MAX_VALUE lets us do the assignment without a warmup
+        properties['acceptable.recovery.lag'] = "9223372036854775807"
+
```

Comment:
Added this configuration to fix the flaky `StreamsOptimizedTest.test_upgrade_optimized_topology`